src/System.Private.CoreLib/shared/System/Diagnostics/DebugProvider.cs

```diff
@@ -33,6 +33,21 @@ public virtual void Write(string message)
                 }
             }
         }
+        
+        public virtual void WriteLine(string message)
+        {
+            Write(message + Environment.NewLine);
+        }
+        
+        public virtual void Indent()
```

Comment:
IndentLevel is virtual already. Do we need really these virtual APIs?

Reply:
I will check

Reply:
You're right. Unindent() and Indent() were not necessary. I removed them.